samples/Java/Basic/ReceiveUsingOffset/src/main/java/com/microsoft/azure/eventhubs/samples/receiveusingoffset/ReceiveUsingOffset.java

```diff
@@ -60,34 +62,19 @@ public static void main(String[] args)
                 receivedEvents = receiver.receiveSync(100);
             }
         } finally {
-            // this is paramount; max number of concurrent receiver per consumergroup per partition is 5
-            receiver.close().whenComplete(new BiConsumer<Void, Throwable>() {
-                public void accept(Void t, Throwable u) {
-                    if (u != null) {
-                        // wire-up this error to diagnostics infrastructure
-                        System.out.println(String.format("closing failed with error: %s", u.toString()));
-                    }
-                    try {
-                        ehClient.closeSync();
-                    } catch (EventHubException sbException) {
-                        // wire-up this error to diagnostics infrastructure
-                        System.out.println(String.format("closing failed with error: %s", sbException.toString()));
-                    }
-                }
-            }).get();
-        }
-    }
+            // cleaning up receivers is paramount;
+            // Quota limitation on maximum number of concurrent receivers per consumergroup per partition is 5
```

Comment:
Maybe:
`cleaning up receivers is paramount as only 5 concurrent receivers are allowed per consumer group per partition`